docs/pages/setup/admin/adding-nodes.mdx

```diff
@@ -134,79 +136,56 @@ auth_service:
 
 ### Start your Node with the invite token and CA pin
 
-<Tabs>
-<TabItem scope={["oss", "enterprise"]} scopeOnly={true} label="Self Hosted">
-
-Execute one of the following commands on a new Node to add it to a cluster.
-Supply the invite token and CA pin you retrieved earlier:
+Execute the following commands on your local machine, then copy the output.
 
 ```code
-$ sudo teleport start \
-   --roles=node \
-   --token=(=presets.tokens.first=) \
-   --ca-pin=(=presets.ca_pin=) \
-   --auth-server=10.12.0.6:3025
+$ echo "export CA_PIN=${CA_PIN?}"
```

Comment:
This feels a bit weird, but I can't really think of another way of doing it other than:
```
echo "export CA_PIN=${CA_PIN?}\nexport INVITE_TOKEN=${INVITE_TOKEN?}"
```

Which feels worse :(

Reply:
Would it make more sense to show the `export` commands the user would run on the Node's host, then let the readers work out for themselves that they should paste the values of the variables from a terminal on their local machine?

Reply:
Perhaps that's easier :) Or a bit less weird.